Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -17,10 +17,6 @@
     <DebugType>Full</DebugType>
   </PropertyGroup>
   
-  <PropertyGroup Condition="$([System.Text.RegularExpressions.Regex]::IsMatch('$(TargetFramework)', '^net\d'))">

```

Comment:
Instead of removing the property group update the regex pattern to `'^net\d{2,}'`